Storyboard Engine/StoryboardObject.py

```diff
@@ -142,12 +146,54 @@ def startLoop(self, start_t, loop_count):
     def add(self, s):
         self.codes.append(s)
 
+
     def printObj(self):
-        self.codes.insert(0, ','.join(
-            [self.type, self.placement, self.alignment, self.file_name, str(self.x), str(self.y)]))
+        self.codes.insert(0, ','.join([self.type, self.placement, self.alignment, self.file_name, str(self.x), str(self.y)]))
         for code in self.codes:
             print(code)
 
+class eventObj():

```

Comment:
Change the class name to **Code**, despite of getList func, also add a function called **getString()**, which can return a string of the Code

Reply:
The getList func return a list of object, so idk override __str__ func maybe useful.
Class name will change. Wait me next commit.